src/v/archival/tests/ntp_archiver_test.cc

```diff
@@ -14,7 +14,6 @@
 #include "bytes/iobuf.h"
```

Comment:
`tests,cloud_storage: adjust existing tests to new api`

for the commit subject, we generally do sub-system first like `cloud_storage/tests: ...`

Reply:
I moved this commit to the one where remote is changed so that tests pass for that commit. will reword other commits according to this guideline